static/js/cohort_details.js

```diff
@@ -327,13 +326,7 @@ require([
         return false;
     });
 
-    // If this is a new cohort, set TCGA Project selected as default
```

Comment:
Is there a reason this was remove? I put it in so that TCGA is selected on page load.